Dockerfile

```diff
@@ -57,6 +61,11 @@ RUN export DEBIAN_FRONTEND=noninteractive && \
     echo "Installing MACS2..." && \
     sudo pip install MACS2==2.1.1.20160309 && \
 
+    echo "Installing ROSE2..." && \
+    # XXX: Current version of ROSE2 is seriously broken. Persuade upstream to
+    # to fix it
+    sudo pip install -e git+git://github.com/jkokosar/rose2.git@a8fe807677681fe63557ffc1e46ade3e98c631e7#egg=rose2
```

Comment:
Package containing the above mentioned commit is now included in the latest PyPi version: https://pypi.python.org/pypi/rose2/1.0.2